src/lib/FlightTasks/tasks/ManualAltitudeSmoothVel/FlightTaskManualAltitudeSmoothVel.cpp

```diff
@@ -60,8 +58,6 @@ void FlightTaskManualAltitudeSmoothVel::reActivate()
 	// The task is reacivated while the vehicle is on the ground. To detect takeoff in mc_pos_control_main properly
 	// using the generated jerk, reset the z derivatives to zero
 	_smoothing.reset(0.f, 0.f, _position(2));
-
-	_initEkfResetCounters();
```

Comment:
Is that case of `reActivate()` covered in the new implementation?
It's now "just" called in `activate()`

Reply:
I don't see why we should re-initialize the counter during the reactivate; I know I put it there before, but I don't see why so I decided to remove it.

Reply:
true